packages/examples/src/relay/Feedback/Feedback.stories.tsx

```diff
@@ -15,14 +15,21 @@ import type { FeedbackStoryQuery } from "./__generated__/FeedbackStoryQuery.grap
 import { getSchema } from "../../testing-utils/getSchema";
 import * as React from "react";
 import type { events } from "../../events/events";
+import { RecordSource, Store } from "relay-runtime";
 
 const schema = getSchema();
 
 const MockPayloadGenerator = new PayloadGenerator(schema);
 
+const novaDecorator = getNovaDecorator(schema, {
+  getEnvironmentOptions: () => ({
+    store: new Store(new RecordSource()),
+  }),
+});
```

Comment:
Is it intentional to keep this example around? Don't need to have it since Relay will create the store if not given, but I could see it as an example of how to override config.

Reply:
I wanted to keep it as a check for having some stories with no environment and some with custom env. That way we have both cases verified